modules/openapi-generator/src/main/resources/dart2/api.mustache

```diff
@@ -61,9 +61,10 @@ class {{{classname}}} {
     {{/allParams}}
 
   {{/hasParams}}
-    final path = r'{{{path}}}'{{#pathParams}}
-      .replaceAll('{' + '{{{baseName}}}' + '}', {{{paramName}}}.toString()){{/pathParams}};
+    // ignore: prefer_const_declarations
```

Comment:
We can not manage all new and old lints spread out in the generator. And we do not know which lints eventually are used. I would advocate against this.
Just ignore generated code from linting in your projects, there is no value in linting it.

Reply:
If this recommendation is alright by others, I can just remove all of the ignores.

Reply:
We should run very restrictive lints on the sample projects and make sure we don't emit the lent warning sin the first place.

Reply:
That I can do also. But I can do it in another pr, what do you say?
(bottom line: remove all ignores and fix the source code so we don't need them)